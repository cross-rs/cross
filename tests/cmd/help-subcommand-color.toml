bin.name = "cross"
args = ["help", "--color", "never"]
status = "failed"
