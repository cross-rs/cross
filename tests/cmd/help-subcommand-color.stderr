[..][cross] note[..]: Falling back to `cargo` on the host.
[..]error:[..] The subcommand [..]'--color'[..] wasn't recognized

USAGE:
    cargo <subcommands>

For more information try [..]--help[..]